Installation failed with no such option: --break-system-packages

Setting it up on EC2 - Amazon Linux 

During installation from source it was failing with below issue.

_no such option: --break-system-packages
make: *** [Makefile:27: install_python_packages] Error 2_

WIth some research found below workaround that worked, i.e. setting up PIP_BREAK_SYSTEM_PACKAGES and remove --break-system-packages from MakeFile.

```
export PIP_BREAK_SYSTEM_PACKAGES=1

# Install the required Python packages
install_python_packages:
	@pip3 install -r requirements.txt

```

Sharing it as issue to retrospect if we can use PIP_BREAK_SYSTEM_PACKAGES as default way in MakeFile.

--- a/Makefile
+++ b/Makefile
@@ -22,9 +22,11 @@ install_extension:
 	@cp $(SQL_FILES) $(EXTENSION_DIR)
 	@cp $(CONTROL_FILE) $(EXTENSION_DIR)
 
+export PIP_BREAK_SYSTEM_PACKAGES=1
+
 # Install the required Python packages
 install_python_packages:
-	@pip3 install --break-system-packages -r requirements.txt
+	@pip3 install -r requirements.txt
 
 test: install_extension
 	@./test.sh

--- a/vm.sh
+++ b/vm.sh
@@ -19,7 +19,8 @@ apt-get install -y --no-install-recommends \
     python3-pip \
     make
 
-pip install --break-system-packages -r /pgai/requirements.txt
+export PIP_BREAK_SYSTEM_PACKAGES=1
+pip install -r /pgai/requirements.txt
 
 cp /pgai/ai--*.sql /usr/share/postgresql/16/extension/
 cp /pgai/ai.control /usr/share/postgresql/16/extension/
@@ -29,7 +30,5 @@ chmod -R go+w /usr/lib/postgresql/16/lib/
 echo "/usr/bin/psql -U postgres -c \"create user ubuntu superuser login password 'ubuntu'\"" | sudo su postgres -
 echo "/usr/bin/psql -U postgres -c \"create database ubuntu owner ubuntu\"" | sudo su postgres -
 EOF
-multipass stop pgai
-multipass mount -t native . pgai:/pgai
-multipass start pgai
+multipass restart pgai
 multipass shell pgai

